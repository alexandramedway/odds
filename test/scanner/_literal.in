"hello"
34.59
true
void
"\"x"
2