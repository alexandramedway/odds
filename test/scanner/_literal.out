STRING_LITERAL
NUM_LITERAL
BOOL_LITERAL
VOID_LITERAL
STRING_LITERAL
NUM_LITERAL
